functions/__snapshots__/default.test.ts.snap

```diff
@@ -302,31 +330,43 @@ exports[`should inject metadata for valid collections 3`] = `
     -->
     <link rel="manifest" href="/manifest.json" />
 
-    <link rel="preload" href="/fonts/Inter-roman.var.woff2" as="font" type="font/woff2" crossorigin />
+    <link rel="preload" href="/fonts/Basel-Book.woff" as="font" type="font/woff" crossorigin />
+    <link rel="preload" href="/fonts/Basel-Medium.woff" as="font" type="font/woff" crossorigin />
 
+   
     <style>
       * {
-        font-family: 'Inter', sans-serif;
+        font-family: 'Basel', sans-serif;
         box-sizing: border-box;
       }
 
       /**
-          Explicitly load Inter var from public/ so it does not block LCP's critical path.
+          Explicitly load Basel var from public/ so it does not block LCP's critical path.
         */
       @font-face {
-        font-family: 'Inter custom';
-        font-weight: 100 900;
+        font-family: 'Basel';
+        font-weight: 535;
+        font-style: normal;
+        font-display: block;
+        font-named-instance: 'Book';
+        src: 
+          url(/fonts/Basel-Medium.woff) format('woff');
+      }
+
+      @font-face {
+        font-family: 'Basel';
+        font-weight: 435;
         font-style: normal;
         font-display: block;
-        font-named-instance: 'Regular';
-        src: url(/fonts/Inter-roman.var.woff2) format('woff2 supports variations(gvar)'),
-          url(/fonts/Inter-roman.var.woff2) format('woff2-variations'),
-          url(/fonts/Inter-roman.var.woff2) format('woff2');
+        font-named-instance: 'Book';
+        src: 
+          url(/fonts/Basel-Book.woff) format('woff');
```

Comment:
can we get a woff2 file?

Reply:
according to Cal the woff we tried was causing glitchy rendered text in FF, but I haven't confirmed that myself.

Reply:
@JFrankfurt I'm up for trying it - when I had woff2 in there that's when you were reporting the pixellated fonts. Can I send you the replacement to test locally?